packages/components/modules/payments/web/types.ts

```diff
@@ -23,23 +23,39 @@ export interface PaymentMethod {
     }
   }
 }
-
+interface DefaultPrice {
+  id: string
+  unitAmount: number
+  currency?: string
+  locale?: string
+}
 export interface Product {
   id: string
   name: string
   images: string[]
-  defaultPrice: {
-    id: string
-    unitAmount: number
-    currency?: string
-    locale?: string
-  }
+  description: string
+  defaultPrice: DefaultPrice
```

Comment:
nice update. I think we can still do for other types like marketingFeature, plan, item